cellprofiler/pipeline.py

```diff
@@ -951,6 +951,12 @@ def rl():
                     ).format(git_hash, pipeline_date)
                     logging.warning(message)
                 else:
+                    # pipeline versions pre-3.0.0 have unpredictable formatting
+                    if pipeline_version == 300:
```

Comment:
I wonder where the conversion to `300`. It would be nice if the full version information were captured (e.g., "3.0.0rc1"). Does Python have any semantic version support, like version comparison?